packages/Config/RectorConfig.php

```diff
@@ -279,6 +279,10 @@ public function cacheDirectory(string $directoryPath): void
         SimpleParameterProvider::setParameter(Option::CACHE_DIR, $directoryPath);
     }
 
+    /**
+     * @info Rector no longer uses compiled container, but Laravel lazy one.
+     * This option is for PHPStan container in case the sys_get_temp_dir() is not available for the current user.
+     */
     public function containerCacheDirectory(string $directoryPath): void
```

Comment:
if config cache merged, this method can be deprecated and removed on 1.0 as `Option::CONTAINER_CACHE_DIRECTORY` no longer be used if cache directory merged.